onmt/models/init.lua

```diff
@@ -0,0 +1,3 @@
+require('onmt.models.Model')
+require('onmt.models.seq2seq')
+require('onmt.models.lm')
```

Comment:
yes good point! Done.
I will see if we can even easily transform train.lua for `seq2seq` and `LM`. By default it will be seq2seq but adding LM (or im2text) as first parameter, and it will load LM/im2text specific code.